src/ontology/mondo-ingest.Makefile

```diff
@@ -571,8 +571,9 @@ $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv: $(TMPDIR)/nord.tsv config/external-conte
 .PHONY: external-content-nord
 external-content-nord: $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
 
-tmp/ordo-subsets.tsv: $(COMPONENTSDIR)/ordo.owl
-	$(ROBOT) query -i $< --query ../sparql/select-ordo-subsets.sparql $@
+tmp/ordo-subsets.tsv:
+	$(MAKE) component-download-ordo.owl
```

Comment:
Ah, I see. This is because the component currently strips away the subset information, I think.

Reply:
What is more important: the component adds additional isa links that are not there in ORDO, which confuses the subset pipeline!